roles/cobbler_profile/tasks/import_distro.yml

```
@@ -0,0 +1,88 @@
+---
+# This profile will do all the work necessary to create a new distro/profile
+# pair in Cobbler.
+
+# Since this profile will be used several times in the same playbook,
+# mention the distro name each time.
+- name: Distro name
+  debug: var=distro_name
```

Should we fail if distro_name is not defined?